src/components/SearchResult/SearchResult.tsx

```diff
@@ -0,0 +1,45 @@
+import styles from './SearchResult.module.css';
+
+export type SearchResultProps = {
+  poster_path: object;
+  name: string;
+  vote_average: number;
+};
+
+function SearchResult({
+  name,
+  poster_path,
+  vote_average,
+}: SearchResultProps): JSX.Element {
+  if (name) {
+    return (
+      <div className={styles.container}>
+        <img
+          src={`https://image.tmdb.org/t/p/w500${poster_path}`}
+          alt="Cover"
```

Comment:
Maybe add the name here? 
```suggestion
          alt=`Cover of ${name}`
```